src/bindgen/parser.rs

```diff
@@ -691,6 +691,12 @@ impl Parse {
         let impl_path = ty.unwrap().get_root_path().unwrap();
 
         for item in items.into_iter() {
+            if let syn::Visibility::Public(_) = item.vis {
```

Comment:
I think these would be nicer with `match`.

Reply:
I literally copied the visibility check as it already is in other places. Should I change all of them for consistency?

Reply:
That'd be great, though I guess we can also change them all together in a followup PR. Your call :)

Reply:
Also it looks like this needs a rebase for #318.

Reply:
I've rebased and fixed conflicts. As for visibility check - I started doing it but got a bit derailed in refactoring whole helper traits part and barely stopped :P

I think I'd rather keep this PR scoped and consistent with rest of code for now.